pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -836,94 +484,32 @@ func handleCreate(ctx *verifierContext, objType string,
 		log.Infof("No PersistImageStatus for %s id %s",
 			config.ImageSha256, config.ImageID)
 	}
-	ok, size := markObjectAsVerifying(ctx, config, &status)
```

Comment:
Based on my comment to @deitch  I realize verifier must move the file to a directory it owns (verifier) before starting to read the file. Hence still need that move in here. Sorry for my confusion.

We could have it move it to the verified directory before doing the sha and signature checks, but that means that after a reboot we don't know if what is in the verified directory has been verified or was in progress to be verified.
So two separate directories controller by the verifier are needed.

Reply:
No, you are right here, @eriknordmark ; verifier owns the "in-process" directory (currently publicly called `verifier/` but soon to be obscured and controlled by verifier) and the "completed" directory (currently publicly called `verified/` but soon to be obscured and controlled by verifier).

Sorry for my confusion as well @adarsh-zededa ; it was a work in progress to figure this out.

The key benefit here comes from:

* verifier doesn't know anything about `downloads/` or any other "standard" dirs; it just is given a file path by volumemgr and told, "go verify this file", which could be anywhere
* volumemgr doesn't know anything about `verifier/` or `verified/` or any other "standard dirs; it just tells verifier, "go verify this file", and receives back, "verification success, here is the verified file, and by the way, you cannot edit that file if you tried".

Reply:
Changed.